atd-vze/src/views/Crashes/Maps/CrashEditCoordsMap.js

```diff
@@ -72,7 +72,7 @@ const CrashEditCoordsMap = ({
     // Reset marker to original coordinates or default fallback
     const originalMarkerCoordinates = {
       latitude: latitude_primary || defaultInitialState.latitude,
-      longitude: longitude_primary || defaultInitialState.latitude,
```

Comment:
😬

Reply:
erm :grimacing: is right lol nice catch mike

Reply:
+1 to 😬 that was my oooopsie. initializing the map way outta Austin! 🔭